pkg/credentials/credentials.go

```diff
@@ -101,6 +101,9 @@ func getAWSPluginImage(dpa *oadpv1alpha1.DataProtectionApplication) string {
 }
 
 func getCSIPluginImage(dpa *oadpv1alpha1.DataProtectionApplication) string {
+	if dpa.Spec.Features != nil && dpa.Spec.Features.EnableDataMover {
+		return common.CSIDataMoverPluginImage
```

Comment:
Please respect unsupportedOverrides of CSIPluginImageKey

Reply:
Is this piece unused? I see the following function not referring to this at all.
```
func (r *DPAReconciler) getDataMoverImage(dpa *oadpv1alpha1.DataProtectionApplication) string {
```

Reply:
Done